docs/index.rst

```diff
@@ -77,12 +77,15 @@ and some of the core features of Read the Docs.
 * **Overview of core features**:
   :doc:`Incoming webhooks </webhooks>` |
   :doc:`/custom_domains` |
+  :doc:`/canonical` |
   :doc:`/versions` |
   :doc:`/downloadable-documentation` |
   :doc:`/hosting` |
   :doc:`/server-side-search` |
-  :doc:`/analytics`
-  :doc:`/pull-requests`
+  :doc:`/analytics` |
+  :doc:`/google-analytics` |
+  :doc:`/pull-requests` |
+  :doc:`/build-notifications`
```

Comment:
I think we should also migrate `User-defined redirects` to core features. Silly that we have it in `Advanced features`, since it's something all users want.

Reply:
Also let me know if you want refactoring suggestions like this, or if they are out of scope/annoying :)

Reply:
Done moving `User-defined Redirects` to core features 👍🏽